rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        (
+          sum(avg_over_time(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
+        )
+        /
+        (
+          sum(avg_over_time(argocd_app_info{namespace="gitops-service-argocd"}[5m]))
+          or
+          sum(argocd_app_info{namespace="gitops-service-argocd"}) * 0
+        ) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Less than 95% of GitOps applications are in synced state: {{ $value }}
+        description: >-
+          The percentage total of all Argo CD applications that are in Synced state is less than 95%.
+        runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/gitops/deploy-from-git-to-k8s.md
+    - alert: GitOpsApplicationNoSyncApps
```

Comment:
Do we still need this alert?
If I recall correctly, we only added it to overcome the state in which we have zero sync app. The current query for the other rule should have it covered now.

Reply:
Ah yes, the first four tests showed that we don't need it too.  The first two tests are the same as the third and fourth.  I'll remove this.

Reply:
Checked in a commit to address three of today's review comments